app/_src/networking/transparent-proxy/reachable-services.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Reachable Services
+content_type: how-to
+---
+
+{% assign Kuma = site.mesh_product_name %}
+{% assign tproxy = site.data.tproxy %}
+
+When the transparent proxy is enabled, {{ Kuma }} automatically configures each data plane proxy to connect with **every other** data plane proxy in the same mesh. This setup ensures broad service-to-service communication but can create issues in large meshes:
+
+- **Increased Memory Usage**: The configuration for each data plane proxy can consume significant memory as the mesh grows.
+- **Slower Propagation**: Even small configuration changes for one service must be propagated to all data planes, which can be slow and resource-intensive.
+- **Excessive Traffic**: Frequent updates generate additional traffic between the control plane and data plane proxies, especially when services are added or changed often.
+
+In real-world scenarios, services usually need to communicate only with a few other services rather than all services in the mesh. To address these challenges, {{ Kuma }} offers the **Reachable Services** feature, allowing you to define only the services that each proxy should connect to. Specifying reachable services helps reduce memory usage, improves configuration propagation speed, and minimizes unnecessary traffic.
```

Comment:
```suggestion
In real-world scenarios, services usually need to communicate only with a few other services rather than all services in the mesh. To address these challenges, {{ Kuma }} offers the **Reachable Services** feature, allowing you to define only the services that each proxy should connect to. Specifying reachable services avoids the performance issue mentioned above
```